Unix/configure

```diff
@@ -1383,40 +1383,47 @@ fi
 ##
 ##==============================================================================
 
+use_heimdal = 0
```

Comment:
Will be. I want to be able to config to use heimdal on other archs besides macos.